doc/bucket-class-crd.md

```diff
@@ -100,22 +110,17 @@ status:
 ```
 
 
-# Example
-
-Here are some examples of the cli/YAML usage and BucketClass CRs for the different bucket-class configurations:
-
-Single tier, single backing-store, placement Spread:
+# Examples
+Single tier, single backing store, Spread placement:
 ```shell
-noobaa -n noobaa bucketclass create placement-bucketclass bc --backingstores bs --placement Spread
+noobaa -n app-namespace bucketclass create placement-bucketclass bc --backingstores bs --placement Spread
```

Comment:
I think it might be worth clarifying that for this to work, noobaa system should be running in the `app-namespace` namespace (even though BucketClass will have no problem running in any namespace).